stl/CMakeLists.txt

```diff
@@ -252,6 +252,12 @@ foreach(header ${HEADERS})
     configure_file("${header}" "${PROJECT_BINARY_DIR}/out/inc/${_header_path}" COPYONLY)
 endforeach()
 

```

Comment:
There appear to be real test failures in `GH_002655_alternate_name_broke_linker`; please investigate.